### DISCUSSION TITLE
Migration to Github Enterprise Cloud - can't archive due to repo stuck in Migration

### DISCUSSION THREAD

**Discussion by AndrewSmith593** [community/community#29957] [Enterprise]
I have migrated a repo from on-prem hosted GHES (Github Enterprise Server)to GHEC (Github Enterprise Cloud), and it is accessible in GHEC.

We are now trying to archive the old GHES repo, but we are unable to because of the message on the repo page that reads

" This repository is currently being migrated. It's locked while the migration is in progress."

I am unable to go to the "Settings" for this repo due to restrictions of my workplace, as I'm not an admin on the repo I believe.

Wondering what needs to be done to Stop the migration (even though it's already done), so that we can archive this repo.

Thanks for your help!

Screenshot of the message:
![image](https://user-images.githubusercontent.com/63675982/185429426-31ed3b93-d42d-48b4-aa8f-c868ce3f3a96.png)

**Comment by djdefi** [comment]
A user with GitHub Enterprise Server Site Admin access can unlock this repository by [going to the Site Administrator page](https://docs.github.com/enterprise-server/admin/configuration/configuring-your-enterprise/site-admin-dashboard) for the repository.

The URL should look like: `https://[hostname]/stafftools/repositories/[owner]/[repo]/admin`

Once there they will see the unlock option under the **Single Repository Lock** section on that page.
